README.md

```diff
@@ -36,15 +36,59 @@ LLM App reads a corpus of documents stored in S3 or locally, preprocesses them,
 
 ## Getting Started
 
-This section provides a general introduction on how to start using the app.
+This section provides a general introduction on how to start using the app. You can run it in different settings:
+
+| Pipeline Mode   | Description                                                                                                                                                                                                                                                                        |
+| --------------- | ---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| `contextful`    | In this mode, the app will index the documents located in the `data/pathway-docs` directory. These indexed documents are then taken into account when processing queries. The pathway pipeline being run in this mode is located at `llm_app/pathway_pipelines/contextful/app.py`. |
+| `contextful_s3` | This mode operates similarly to the contextful mode. The main difference is that the documents are stored and indexed from an S3 bucket, allowing the handling of a larger volume of documents. This can be more suitable for production environments.                             |
+| `contextless`   | This pipeline calls OpenAI ChatGPT API but does not use an index when processing queries. It relies solely on the given user query.                                                                                                                                                |
+| `local`         | This mode runs the application using Huggingface Transformers, which eliminates the need for external APIs. It provides a convenient way to use state-of-the-art NLP models locally without additional services.                                                                   |
```

Comment:
```suggestion
| `local`         | This mode runs the application using Huggingface Transformers, which eliminates the need for the data to leave the machine. It provides a convenient way to use state-of-the-art NLP models locally without additional services.                                                                   |
```
or "the need for external services" and then remove "without additional services"?
I would say that HF can be thought of as a kind of an external API (I know it doesn't send any data, but is not created by pathway).